packages/fxa-auth-server/lib/oauth/db/accessToken.js

```diff
@@ -63,6 +63,14 @@ class AccessToken {
     return this.expiresAt.getTime() - Date.now();
   }
 
+  get lastAccessTime() {
+    return this.createdAt;
+  }
+
+  get id() {
+    return this.clientId;
+  }
```

Comment:
aliases to keep some parts of the code happy with mixed `AccessToken` and mysql results